src/java.desktop/share/classes/javax/swing/plaf/metal/MetalBorders.java

```diff
@@ -223,10 +245,21 @@ public static class InternalFrameBorder extends AbstractBorder implements UIReso
          * Constructs a {@code InternalFrameBorder}.
          */
         public InternalFrameBorder() {}
+        /**
+         * Round the double to nearest integer, make sure we round
+         * to lower integer value for 0.5
+         *
+         * @param d number to be rounded
+         * @return a {@code int} which is the rounded value of provided number
+         */
+        private static int roundDown(double d)
```

Comment:
The name of this method is a bit misleading, the operation doesn't actually round down, but rounds the number normally except for the x.5 case right? Does this operation exist somewhere else?

Reply:
Yes, that is correct. The method doc does clarify the purpose. But I see your point - probably `roundOff()` would sound less ambiguous and neutral?

Reply:
[`roundHalfDown`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/RoundingMode.html#HALF_DOWN)?

Reply:
@stanio Thank you for the suggestion- roundHalfDown fits the functionality I was looking for.

Reply:
Since you'll be changing the name of the method, please place the opening brace on the same line as the method declaration.